cms/utils/copy_plugins.py

```diff
@@ -7,20 +7,33 @@ def copy_plugins_to(old_plugins, to_placeholder,
     """
     Copies a list of plugins to a placeholder to a language.
     """
+    # import pdb; pdb.set_trace()
     # TODO: Refactor this and copy_plugins to cleanly separate plugin tree/node
     # copying and remove the need for the mutating parameter old_parent_cache.
     old_parent_cache = {}
-    # For subplugin copy, first plugin's parent must be nulled before copying.
+    # For subplugin copy, top-level plugin's parent must be nulled
+    # before copying.
     if old_plugins:
-        old_plugins[0].parent = old_plugins[0].parent_id = None
-    new_plugins = [old.copy_plugin(to_placeholder, to_language or old.language,
-                                   old_parent_cache, no_signals)
-                   for old in old_plugins]
+        old_parent = old_plugins[0].parent
```

Comment:
is ``id(old_parent) != old_plugins[0].parent``? Otherwise you would override old_parent in the first forloop iteration.
If that is the case maybe consider using ``for old_plugin in old_plugins[1:]`` instead ``for old_plugin in old_plugins``